package.json

```diff
@@ -30,7 +30,8 @@
   "devDependencies": {
     "husky": "^7.0.0",
     "jest": "^27.5.1",
-    "jest-extended": "^2.0.0"
+    "jest-extended": "^2.0.0",
+    "jest-sorted": "^1.0.14"
   },
   "jest": {
     "setupFilesAfterEnv": [
```

Comment:
Add jest-sorted to this array and then that should fix your "not a function" problem.